site/docs/quarks-getting-started.md

```diff
@@ -25,9 +25,13 @@ Quarks' primary API is functional where streams are sourced, transformed, analyz
 
 ### Downloading Apache Quarks
 
-To use Quarks, access the source code and build it. You can read more about building Quarks [here](https://github.com/apache/incubator-quarks/blob/master/DEVELOPMENT.md).
+To use Quarks, you can either download the source code and build it yourself, or you can download an install a pre-built release. 
 
-After you build the Quarks package, you can set up your environment.
+You can read more about building Quarks [here](https://github.com/apache/incubator-quarks/blob/master/DEVELOPMENT.md).
+
+You can find pre-built releases [here](https://github.com/quarks-edge/quarks/releases).
```

Comment:
I think it would be a good idea to add a note here that these are not Apache releases.